ui-participant/src/hub/StudyResearchTasks.tsx

```diff
@@ -61,9 +61,9 @@ export default function StudyResearchTasks(props: StudyResearchTasksProps) {
             className="btn rounded-pill ps-4 pe-4 fw-bold btn-primary"
           >
             {enrolleeHasStartedTaskType(enrollee, nextTask.taskType)
-              ? 'Continue'
-              : 'Start'}
-            {' '}{taskTypeDisplayMap[nextTask.taskType]}
+              ? i18n('taskContinue')
+              : i18n('taskStart')}
+            {' '}{i18n(taskTypeMap[nextTask.taskType])}
```

Comment:
Probably not, but I am slightly worried that some languages might flip this around.

Reply:
Yeah, some of these may need to become full phrases instead of appending bits of strings. Pluralization is another area that will need to be treated carefully